content-src/components/ActivityFeed/ActivityFeed.scss

```diff
@@ -1,9 +1,14 @@
 .activity-feed {
+  border-bottom: solid 1px $faintest-black;
   font-size: 13px;
   line-height: 1.4;
   list-style: none;
   margin: 0;
   padding: 0;
+
+  &:last-child {
```

Comment:
This isn't working because the `ul` is nested inside a div, you need to either apply it to the container or change the structure